### PR TITLE
pkg/docker-ce: add vpnkit-expose-port

### PR DESCRIPTION
**Pull request by MagnusS**
Adds vpnkit-expose-port to the docker-ce image.

vpnkit-expose-port is the dockerd userland proxy used to expose
forwarding ports with vpnkit. This adds the binary to the image in
/usr/bin/vpnkit-expose-port, but does not enable it by default.

Signed-off-by: Magnus Skjegstad <magnus@skjegstad.com>
ralph-nhspca-2926

### CODE REVIEW

File: pkg/docker-ce/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+FROM linuxkit/vpnkit-forwarder:883de832c2c3cb72cd9b01e3f7bd788649e0f2c2 AS vpnkit
```
Comment (justincormack) [inline pkg/docker-ce/Dockerfile]: I think its more logical to add this just before `FROM scratch` so we consistently do the `FROM linuxkit/alpine` at the top of files.

Comment (MagnusS) [inline pkg/docker-ce/Dockerfile]: Ok - moved and updated

**Comment by justincormack** [comment]
This needs to update the `docker-ce` hash in the examples that use it once its pushed

**Comment by MagnusS** [comment]
@riyazdf thanks!

yml files updated

**Comment by MagnusS** [comment]
Ah missed one - should be all now

**Comment by riyazdf** [comment]
@MagnusS: thanks!  I think there's also one more usage in `projects/kubernetes/image-cache/Dockerfile`

**Comment by MagnusS** [comment]
@riyazdf that was the one I missed - should be in the last update

**Comment by riyazdf** [comment]
👍  needed a refresh!  All good :)

**Pull request merged**